### PR TITLE
Increase  thumbnail on package detail page and center image

### PR DESCRIPTION
**Pull request by szakarias**
Increase screenshot thumbnail size on the package detail page and center image in thumbnail if the ratio is not 1:1. The collections icon will stay in the bottom right corner.

### CODE REVIEW

**Comment by szakarias** [comment]
image

**Comment by szakarias** [comment]
[cast.webm](https://user-images.githubusercontent.com/3865266/191954049-413c2774-7fd7-4ee1-aa81-a688c5980780.webm)

File: pkg/web_css/lib/src/_list.scss [hunk 1]
```diff
@@ -128,15 +128,26 @@
...
-  flex-grow: 0;
-  flex-shrink: 0;
-  flex-basis: 30px;
+  height: 30px;
   width: 30px;
-  align-self: flex-end;
-  margin-left: -30px;
   opacity: 70%;
   background-color: #aeaeae;
+  margin-top: auto;
```
Comment (isoos) [inline pkg/web_css/lib/src/_list.scss:140]: maybe a `position: relative; left: -30px` could be used here, but I'm fine with this too

**Review by isoos — APPROVED** [review]

File: pkg/web_css/lib/src/_list.scss (hunk 1 above)
Comment (szakarias) [reply pkg/web_css/lib/src/_list.scss:140]: I cannot make it work with that, so I'll keep this since you are okay with this too.

**Pull request merged**